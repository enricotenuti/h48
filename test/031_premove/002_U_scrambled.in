U
UL0 BL0 BR1 DL0 FR0 DF0 DB1 DR1 UB0 FL0 UF0 UR1 DFL0 UFR1 DBR1 UBR2 DBL2 DFR0 UFL1 UBL2

// Scramble: U R D' L D' F L2 D L F B D2 B' L2 F U2 L2 D2 R2 L2 B'
