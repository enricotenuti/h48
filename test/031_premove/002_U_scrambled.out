UB0 BL0 BR1 DL0 FR0 DF0 DB1 DR1 UR0 FL0 UL0 UF1 DFL0 UFL1 DBR1 UFR2 DBL2 DFR0 UBL1 UBR2
